### PR TITLE
fix(RTE): lic serialization is wrong

### PR DESCRIPTION
**Pull request by fulopdaniel**
https://app.clickup.com/t/2523021/TASK-10428

### CODE REVIEW

File: packages/fondue/src/components/RichTextEditor/serializer/utils/serializeNodeToHtmlRecursive.ts [hunk 1]
```diff
@@ -126,7 +126,7 @@ const MapNodeTypesToHtml: { [key: string]: ({ ...args }: Arguments) => string }
             getLiStyles(node, styles),
```
Comment (ragi96) [inline packages/fondue/src/components/RichTextEditor/serializer/utils/serializeNodeToHtmlRecursive.ts:126]: Also needs to be fixed within the checklist serialization

File: packages/fondue/src/components/RichTextEditor/serializer/utils/serializeNodeToHtmlRecursive.ts [hunk 2]
```diff
@@ -126,7 +126,7 @@ const MapNodeTypesToHtml: { [key: string]: ({ ...args }: Arguments) => string }
...
         )}">${children}</li>`,
     [ELEMENT_LIC]: ({ classNames, children, node }) =>
-        `<p dir="auto" class="${classNames} ${getLicElementClassNames(node)}"><span>${children}</span></p>`,
+        `<p dir="auto" class="${classNames} ${getLicElementClassNames(node)}"><span class="tw-min-w-0">${children}</span></p>`,
```
Comment (ragi96) [inline packages/fondue/src/components/RichTextEditor/serializer/utils/serializeNodeToHtmlRecursive.ts:129]: Better to put the classes into a variable and use it in both places (plugin and serializer)

https://github.com/Frontify/fondue/blob/79abb46358c17ca2f750f3b76b81ac4fe0d887d6/packages/fondue/src/components/RichTextEditor/Plugins/ListPlugin/ListItemContentMarkupElement.tsx#L21

File: packages/fondue/src/components/RichTextEditor/serializer/utils/serializeNodeToHtmlRecursive.ts (hunk 1 above)
Comment (fulopdaniel) [reply packages/fondue/src/components/RichTextEditor/serializer/utils/serializeNodeToHtmlRecursive.ts:129]: Thanks, fixed

File: packages/fondue/src/components/RichTextEditor/serializer/utils/serializeNodeToHtmlRecursive.ts (hunk 2 above)
Comment (fulopdaniel) [reply packages/fondue/src/components/RichTextEditor/serializer/utils/serializeNodeToHtmlRecursive.ts]: Thanks, adjusted

File: .changeset/sharp-humans-talk.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
+"@frontify/fondue": patch
+---
+
+fix(RTE): lic serialization is wrong
```
Comment (ragi96) [inline .changeset/sharp-humans-talk.md:5]:
```suggestion
fix(RTE): lic and checkbox item serialization
```

**Review by ragi96 — APPROVED** [review]
Approved but can you please adjust the changeset

**Pull request merged**